Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -137,11 +143,6 @@ struct FloatingPanel<Content>: View where Content: View {
                 let deltaY = $0.location.y - (latestDragGesture?.location.y ?? $0.location.y)
                 let proposedHeight = height + ((isCompact ? -1 : +1) * deltaY)
                 
-                if shouldSendResign {
-                    UIApplication.shared.sendAction(#selector(UIResponder.resignFirstResponder), to: nil, from: nil, for: nil)
-                    shouldSendResign = false
-                }
```

Comment:
Dismissing the keyboard while the user is still dragging can create an issue where the handle moves away from the user's finger. Now we just wait for the gesture to end.